calcom/cal.com
# feat: travel schedules to schedule timezone changes

## What does this PR do?

New feature to add travel schedules for scheduling a timezone change. 

Loom showing availability with travel schedules: https://www.loom.com/share/29d8500385fb4ff78e025021ad5710ea

Fixes #6790

## Type of change

- New feature (non-breaking change which adds functionality)

## How should this be tested?

To run locally change your system timezone to utc time.

- Create travel schedules 
- test availability 
- test CRON API endpoint `/api/cron/changeTimeZone`
    - change system timezone to right after travel schedule started (midnight of user timezone)
    - check if timezone changed & check if timezone of default schedule changed
